### PR TITLE
update VCS and postgres PL docs for accuracy and completeness

### PR DESCRIPTION
**Pull request by dhaworth**
## What are you changing in this pull request and why?

Noticed an incorrect principal ARN and some missing information in a couple of the PrivateLink docs, so updating them accordingly. In summary:
- Postgres PL - Separate out connection request acceptance into separate step
- VCS PL
    - Fix incorrect principal ARN
    - Change to numbered headers
    - Add missing step for submitting support request
    - Clarify DNS settings 

## Checklist
- [X] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- NA [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [X] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] Review from docs team for style and quality
- [x] Review from Enterprise for content

### CODE REVIEW

**Review by aaronbini — APPROVED** [review]
Mostly reviewed to check for typos (there weren't any!), and I trust that these are the right changes since Dave is an expert in this area.

File: website/docs/docs/cloud/secure/postgres-privatelink.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ Subject: New Multi-Tenant PrivateLink Request
 
 dbt Labs will work on your behalf to complete the PrivateLink setup. Please allow 1-2 business days for this process to complete. Support will contact you when the endpoint is available.
 
+### 5. Accepting the connection request
+
+When you have been notified that the resources are provisioned within the dbt Cloud environment, you must accept the endpoint connection (unless the VPC Endpoint Service is set to auto-accept connection requests). Requests can be accepted through the AWS console, as seen below, or through the AWS CLI.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/accept-request.png" width="80%" title="Accept the connection request" />
```
Comment (matthewshaver) [inline website/docs/docs/cloud/secure/postgres-privatelink.md:75]: Is the endpoint ID in this doc sensitive - does it need to be obfuscated or is it ok as-is?

File: website/docs/docs/cloud/secure/vcs-privatelink.md [hunk 2]
```diff
@@ -46,23 +49,38 @@ You need these resource to create a PrivateLink connection, which allows the dbt
     - **Load balancer:** Attach the NLB created in the previous step.
     - **Acceptance required (recommended)**:  When enabled, requires a new connection request to the VPC Endpoint Service to be accepted by the customer before connectivity is allowed ([details](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#accept-reject-connection-requests)).
 
-    Once these resources have been provisioned, access needs to be granted for the dbt Labs AWS account to create a VPC Endpoint in our VPC. On the newly created VPC Endpoint Service, add a new [Allowed Principal](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permissions) for the appropriate dbt Labs principal: 
+ ### 2. Grant dbt AWS account access to the VPC Endpoint Service
```
Comment (matthewshaver) [inline website/docs/docs/cloud/secure/vcs-privatelink.md]: Spacing is off in the preview. Hoping this fixes it
```suggestion
### 2. Grant dbt AWS account access to the VPC Endpoint Service
```

File: website/docs/docs/cloud/secure/vcs-privatelink.md [hunk 3]
```diff
@@ -46,23 +49,38 @@ You need these resource to create a PrivateLink connection, which allows the dbt
...
 
-To complete the connection, dbt Labs must now provision a VPC Endpoint to connect to your VPC Endpoint Service. This requires you send the following information:
- 
- - VPC Endpoint Service name:
+Once the VPC Endpoint Service is provisioned and configured find the service name in the AWS console by navigating to **VPC** → **Endpoint Services** and selecting the appropriate endpoint service. Copy the service name field value and include it in your communication to dbt Cloud support.
 
  <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/vpc-endpoint-service-name.png" width="80%" title="Location of the VPC Endpoint Service name in the AWS console" />
 
- - **DNS configuration:** If the connection to the VCS service requires a custom domain and/or URL for TLS, a private hosted zone can be configured by the dbt Labs Infrastructure team in the dbt Cloud private network. For example:
+ ### NOTE: Custom DNS configuration
```
Comment (matthewshaver) [inline website/docs/docs/cloud/secure/vcs-privatelink.md]: We don't style `Note:` in header formats. Is it ok if I change this to a callout?

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**